app/src/main/java/com/breadwallet/presenter/entities/BRSettingsItem.java

```diff
@@ -4,16 +4,21 @@
 
 public class BRSettingsItem {
 
-    public boolean isSection;
+    public enum SettingItemsType {
+        SECTION, ADDON, SWITCH
+    }
+
+    public SettingItemsType type;
     public String title;
     public String addonText;
+    public Boolean isChecked;
     public View.OnClickListener listener;
 
```

Comment:
This is a really good start. We need to make sure it's visible.  Will you add this:
- Status label
- Switch background color

| Settings Dev Mode: On | Settings Dev Mode: Off |
|--------------------|-------------------|
|![Developer Mode_ Switch](https://user-images.githubusercontent.com/2899463/213271548-0b80cb10-f0a3-4004-bf5f-146ea8a2bd92.png)|![Developer Mode_ Switch Off](https://user-images.githubusercontent.com/2899463/213271581-78978bdf-ab29-4612-ba9c-14b1fa31abb1.png)|

Reply:
How does this look? 
<img width=300 src=https://user-images.githubusercontent.com/392399/213830659-ea4b78a1-d600-4c58-a304-b96d848617e5.png />
<img width=300 src=https://user-images.githubusercontent.com/392399/213830662-b7a650c2-d101-4d09-8000-9ba143eb1c4d.png />

Reply:
✅